.activate-venv.sh

```diff
@@ -21,5 +21,5 @@ fi
 # 'ARCHFLAGS' fixes a problem with OSX refusing to compile a dependency
 export ARCHFLAGS="-Wno-error=unused-command-line-argument-hard-error-in-future"
 
-source venv/bin/activate
+. ./venv/bin/activate
```

Comment:
so (iirc) `source` is a *bash* language construct! if you do a `which source` it won't be able to find it on most systems. the `.` in `sh` has the same semantics and both `bash` and `sh` support the  `.`. I also recall Graham having a problem with `source` once upon a time on windows, but windows isn't really supported (yet).